infrastructure/nomad/playbooks/deploy.yml

```diff
@@ -495,7 +495,7 @@
             echo "${ALLOC}" | jq --arg address "0x${ADDRESS}" '
               . + {
                 ($address): {
-                    "balance": "0x10000000000000000000000000000"
+                    "balance": "0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD4A51000FDA0FFFF"
```

Comment:
Does this match `DEPLOYER_GENESIS_ALLOCATION` from `contracts/scripts/standard-bridge/DeployStandardBridge.s.sol`?

Reply:
Yes, otherwise it would fail when contracts are deployed.